db/seeds.rb

```diff
@@ -35,6 +35,7 @@ def load_base_file(file)
   "add_contracts.rb",
   "add_declarations.rb",
   "add_api_tokens.rb",
+  "process_statements.rb",
 ].each do |seed_file|
   Rails.logger.info("seeding #{seed_file}")
   load_base_file(seed_file)
```

Comment:
On my old slow laptop, wrapping this in a transaction drops `time rails db:schema:load db:seed`  from 2:57 to 1:43

```ruby
ApplicationRecord.transaction do
  load_base_file(seed_file)
end
```

Reply:
not vital but seems like a quick win

Reply:
just re-run this testing locally and speed up is reproducible